core/src/com/unciv/logic/city/City.kt

```diff
@@ -238,11 +250,9 @@ class City : IsPartOfGameInfoSerialization, INamed {
     }
 
     fun getReserve(stat: GameResource): Int {
-        if (stat is TileResource && stat.isCityWide) {
-            return if (stat.isStockpiled) {
-                // TODO
-                0
-            } else 0
+        if (stat is TileResource) {
+            if (!stat.isStockpiled) return 0
+            if (stat.isCityWide) return resourceStockpiles[stat.name]
```

Comment:
What do we do when it's civ-wide? Fall down to "civ.getReserve(stat)" below?
I'd rather this was explicit in the "stat is TileResource", that it become a "return when()" so we see that we treat all TileResources cases exhaustively

Reply:
I had it like that originally, but wasn't sure if it should be explicit